pkg/clients/cliServiceClient.go

```diff
@@ -130,8 +130,10 @@ type Metadata struct {
 }
 
 type ClusterContext struct {
-	WebhookVersion string `json:"webhookVersion"`
-	IsInCluster    bool   `json:"isInCluster"`
+	WebhookVersion  string  `json:"webhookVersion"`
+	IsInCluster     bool    `json:"isInCluster"`
+	NodesCount      *int    `json:"nodesCount"`
+	NodesCountError *string `json:"nodesCountError"`
```

Comment:
Just a thought, if it's good to use `*` for these properties, maybe we should do it for all?

Reply:
the idea behind this is that when it translates to json, I wanted it to be `null` and not 0 or empty string